src/com/trilead/ssh2/packets/PacketUserauthTokenGssapiWithMic.java

```diff
@@ -0,0 +1,104 @@
+package com.trilead.ssh2.packets;
+
+import com.trilead.ssh2.auth.GSSContextKrb5;
+
+public class PacketUserauthTokenGssapiWithMic {
+	
+	private static final String GSSAPI_WITH_MIC = "gssapi-with-mic";
+	private static final String SSH_CONNECTION = "ssh-connection";
+	private String user;
+	private String host;
+	private byte[] payload;
+	GSSContextKrb5 context = null;
+
+	public PacketUserauthTokenGssapiWithMic(String user, String host) {
+		
+		this.user = user;
+		this.host = host;
+		
+		context = new GSSContextKrb5();
+		
+		try 
+		{
+			context.create(this.host);
+		} catch (Exception e) 
+		{
+			e.printStackTrace();
```

Comment:
Use a logger if you need to show the exception contents. Also don't catch Exception unless the underlying class specifically throws it.